### PR TITLE
expose node-registry-url argument

### PR DESCRIPTION
**Pull request by poulet42**
This would allow the action consumers to pass a custom registry-url (and set up auth)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -107,6 +107,7 @@ runs:
       with:
         cache: 'pnpm'
         node-version: ${{ inputs.node-version }}
+        registry-url: ${{ inputs.node-registry-url }}
```
Comment (NullVoxPopuli) [inline action.yml:114]: At the veeery top of this file, can you add a description for this new input? Like node-version has?

Thanks!

**Comment by NullVoxPopuli** [comment]
Thanks for this! I'll test it out today and get it released for ya

File: action.yml (hunk 1 above)
Comment (poulet42) [reply action.yml:114]: Oops ! I added the input entry :)

**Comment by NullVoxPopuli** [comment]
Or, if you have a sample action log i can look at seeing that this works, i can release sooner! 🎉

Things i need to see 👍 
- same behavior with new option omitted 
- option used uses the registry

Thanks again!

**Comment by poulet42** [comment]
> * same behavior with new option omitted -> defaults to `https://registry.npmjs.org/`
https://github.com/poulet42/action-setup-pnpm/actions/runs/5163869486/jobs/9302431750

> * option used uses the registry -> use the provided "node-registry-url": `https://registry.npmmirror.com/`
https://github.com/poulet42/action-setup-pnpm/actions/runs/5163856271/jobs/9302412758

🎉

**Review by NullVoxPopuli — APPROVED** [review]

**Comment by NullVoxPopuli** [comment]
https://github.com/NullVoxPopuli/action-setup-pnpm/releases/tag/v2.2.0

Should also be on `v2` as well

**Pull request merged**